src/views/dependencyExplorer.ts

```diff
@@ -137,17 +140,28 @@ export class DependencyExplorer implements Disposable {
             return;
         }
 
+        if (this._revealingUri === uri.toString()) {
+            return;
+        }
+        this._revealingUri = uri.toString();
+
         let node: DataNode | undefined = explorerNodeCache.getDataNode(uri);
         if (!node) {
             const paths: INodeData[] = await Jdtls.resolvePath(uri.toString());
-            if (!paths || paths.length === 0) {
-                return;
+            if (!_.isEmpty(paths)) {
+                node = await this._dataProvider.revealPaths(paths);
             }
-            node = await this._dataProvider.revealPaths(paths);
         }
 
-        if (this._dependencyViewer.visible) {
```

Comment:
Is `this._dependencyViewer.visible` will be always `ture`? Or it should not be a condition for `reveal`?

Reply:
According to the document for the `reveal()` API:

> Reveals the given element in the tree view. If the tree view is not visible then the tree view is shown and element is revealed.